property/index.js

```diff
@@ -115,54 +118,128 @@ module.exports = yeoman.generators.Base.extend({
       {
          name: 'defaultValue',
          message: 'Default value[leave blank for none]:',
-         default: null
+         default: null,
+         when: function(answers) {
+          return answers.type !== null && 
+            answers.type !== 'buffer'&&
```

Comment:
Style: add a space character before `&&`. Same for the next line.

_(we should add eslint to this repository, it would catch issues like this one)_